spacy/pipeline/tagger.pyx

```diff
@@ -90,6 +90,16 @@ class Tagger(Pipe):
         """
         return tuple(self.cfg["labels"])
 
+    @property
+    def label_data(self):
+        """Data about the labels currently added to the component.
+
+        RETURNS (Dict): The labels data.
+
+        DOCS: https://nightly.spacy.io/api/tagger#labels
```

Comment:
The links to the DOCS refers to `labels` instead of `label_data` (also for some other components)